pkg/nodeagent/test/storefactory.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"centaurusinfra.io/fornax-serverless/pkg/nodeagent/store"
+	"centaurusinfra.io/fornax-serverless/pkg/nodeagent/store/factory"
+	"centaurusinfra.io/fornax-serverless/pkg/nodeagent/store/sqlite"
+	_ "github.com/mattn/go-sqlite3"
+	v1 "k8s.io/api/core/v1"
+)
+
+func main() {
```

Comment:
how is this test (main) invoked? is this a manual test case?

Reply:
yes, I have not make unit test, thought this store is simply put/get object, better to test it with real data.